ctapipe/containers.py

```diff
@@ -1143,3 +1195,74 @@ class ArrayEventContainer(Container):
         default_factory=MonitoringContainer,
         description="container for event-wise monitoring data (MON)",
     )
+
+
+class SchedulingBlockContainer(Container):
+    """Stores information about the scheduling block. This is a simplified version
+    of the SB model, only storing what is necessary for analysis.
+
+    References
+    ----------
+    .. [sb-ob] Oya et al, Scheduling Block Data Model Specification,
+        CTA-SPE-COM-000000-0003, Issue 1, Rev. c
+    """
+
+    default_prefix = ""
+    sb_id = Field(-1, "Scheduling block ID", type=np.int64)
+    sb_type = Field(
+        SchedulingBlockType.UNKNOWN,
+        description="Type of scheduling block",
+        type=SchedulingBlockType,
+    )
+    observing_mode = Field(
+        ObservingMode.UNKNOWN,
+        "Defines how observations within the Scheduling Block are distributed in space",
+        type=ObservingMode,
+    )
+    pointing_mode = Field(
+        PointingMode.UNKNOWN, "Defines how the telescope drives move", type=PointingMode
+    )
+
+
+class ObservationBlockContainer(Container):
+    """Stores information about the observation"""
+
+    default_prefix = ""
+    obs_id = Field(None, "Observation Block ID", type=np.int64)
```

Comment:
same here, dl0 data model has uint64. I'd also use a default of `np.iinfo(np.uint64).max` so the column is always written.